MergePythonSDK/hris/model/employee.py

```diff
@@ -428,40 +448,40 @@ def __init__(self, *args, **kwargs):  # noqa: E501
-        self.date_of_birth = kwargs.get("date_of_birth", None)
-        self.hire_date = kwargs.get("hire_date", None)
-        self.start_date = kwargs.get("start_date", None)
-        self.remote_created_at = kwargs.get("remote_created_at", None)
-        self.employment_status = kwargs.get("employment_status", None)
-        self.termination_date = kwargs.get("termination_date", None)
-        self.avatar = kwargs.get("avatar", None)
-        self.custom_fields = kwargs.get("custom_fields", None)
+        self.remote_id: Optional[str, none_type] = kwargs.get("remote_id", None)
+        self.employee_number: Optional[str, none_type] = kwargs.get("employee_number", None)
+        self.company: Optional[str, none_type] = kwargs.get("company", None)
+        self.first_name: Optional[str, none_type] = kwargs.get("first_name", None)
+        self.last_name: Optional[str, none_type] = kwargs.get("last_name", None)
+        self.display_full_name: Optional[str, none_type] = kwargs.get("display_full_name", None)
+        self.username: Optional[str, none_type] = kwargs.get("username", None)
+        self.groups: Optional[List[str, none_type]] = kwargs.get("groups", list())
+        self.work_email: Optional[str, none_type] = kwargs.get("work_email", None)
+        self.personal_email: Optional[str, none_type] = kwargs.get("personal_email", None)
+        self.mobile_phone_number: Optional[str, none_type] = kwargs.get("mobile_phone_number", None)
+        self.employments: Optional[List[str, none_type]] = kwargs.get("employments", list())
+        self.home_location: Optional[str, none_type] = kwargs.get("home_location", None)
+        self.work_location: Optional[str, none_type] = kwargs.get("work_location", None)
+        self.manager: Optional[str, none_type] = kwargs.get("manager", None)
+        self.team: Optional[str, none_type] = kwargs.get("team", None)
+        self.pay_group: Optional[str, none_type] = kwargs.get("pay_group", None)
+        self.ssn: Optional[str, none_type] = kwargs.get("ssn", None)
+        self.gender: Optional[bool, date, datetime, dict, float, int, list, str, none_type] = kwargs.get("gender", None)
+        self.ethnicity: Optional[bool, date, datetime, dict, float, int, list, str, none_type] = kwargs.get("ethnicity", None)
+        self.marital_status: Optional[bool, date, datetime, dict, float, int, list, str, none_type] = kwargs.get("marital_status", None)
+        self.date_of_birth: Optional[datetime, none_type] = kwargs.get("date_of_birth", None)
+        self.hire_date: Optional[datetime, none_type] = kwargs.get("hire_date", None)
+        self.start_date: Optional[datetime, none_type] = kwargs.get("start_date", None)
+        self.remote_created_at: Optional[datetime, none_type] = kwargs.get("remote_created_at", None)
+        self.employment_status: Optional[bool, date, datetime, dict, float, int, list, str, none_type] = kwargs.get("employment_status", None)
```

Comment:
This type feels off. Is it just the case that enums are generally wonky? Why not just `str`?